test/terra/backends/qasm_simulator/qasm_standard_gates.py

```diff
@@ -79,6 +80,14 @@
     (U3Gate, 3),
     (UGate, 3)
 ]
+BASIS_GATES = [
```

Comment:
Maybe we should update this to just be the old and new IBMQ backend basis gate lists to reduce test load:

```python
BASIS_GATES = [
    None,
    ['id', 'u1', 'u2', 'u3', 'cx'],
    ['id', 'rz', 'sx', 'x', 'cx']
]
```